browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -920,7 +920,17 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		if (data.command) {
 			var iconParent = expander.children('.ui-expander').get(0);
 			var icon = L.DomUtil.create('div', 'ui-expander-icon-right ' + builder.options.cssClass, iconParent);
-			builder._controlHandlers['toolitem'](icon, {type: 'toolitem', command: data.command, icon: builder._createIconURL('morebutton')}, builder);
+			builder._controlHandlers['toolitem'](
+				icon,
+				{
+					type: 'toolitem',
+					command: data.command,
+					aria : {
+						label: (data.children[0].text) ? 'More options for ' + data.children[0].text : '',
```

Comment:
Hmm - surely for such things we should translate the text, and if we are translating it we should not concatenate strings like that, but do some %1 type substitution into a translated string using a stock helper (?).

Reply:
you can translate string with: `_('some string')`